packages/legend-application-studio/src/components/editor/side-bar/ProjectOverview.tsx

```diff
@@ -19,46 +19,80 @@ import { observer } from 'mobx-react-lite';
 import { LEGEND_STUDIO_TEST_ID } from '../../LegendStudioTestID.js';
 import {
   clsx,
+  AssistantIcon,
   CustomSelectorInput,
+  ExternalLinkIcon,
   ShareIcon,
+  Dialog,
   PanelLoadingIndicator,
-  ContextMenu,
-  SyncIcon,
   PencilIcon,
-  InfoCircleIcon,
+  SyncIcon,
   TimesIcon,
-  UsersIcon,
-  UserIcon,
-  ExternalLinkIcon,
-  Dialog,
+  InfoCircleIcon,
   PanelContent,
   Modal,
   ModalBody,
   ModalFooter,
+  ContextMenu,
+  UserIcon,
+  UsersIcon,
+  PanelHeader,
+  PanelHeaderActionItem,
+  PanelHeaderActions,
+  PanelFormTextEditor,
+  PanelForm,
+  PanelSection,
+  Button,
+  Panel,
 } from '@finos/legend-art';
-import { PROJECT_OVERVIEW_ACTIVITY_MODE } from '../../../stores/sidebar-state/ProjectOverviewState.js';
+import { useParams } from 'react-router';
```

Comment:
let's use the `useParams` from `@finos/legend-application`, we're trying to `abstract away` `react-router`